front/app/modules/commercial/content_builder/admin/components/ContentBuilderSettings/index.tsx

```diff
@@ -59,6 +63,7 @@ const ContentBuilderSettings = () => {
       {selected.settings && React.createElement(selected.settings)}
       {selected.isDeletable ? (
         <Button
+          w="120px"
```

Comment:
It's generally a bad practice to add fixed width to buttons or other elements - you don't know how big it will be in each language. Please remove 🙏

Reply:
Ah that makes sense. In the prototype the button is not the full width of the Settings panel - what would be the recommended way to control the width of the button?

Reply:
@IvaKop Just checking in again regarding this? :)

Reply:
Ah, sorry! If you put it in a flex container it should adjust according to the length of the text. But even if we leave it as it - it's fine :)